web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
I see more impossible to have 50 classes in a file, or several files with a weird grouping classification. The exception are models like sale.order.line or other one2many models that can go in the same file as sale.order.

We can't debate again and again each question. Please follow the already established rules.